packages/scaffold/src/views/w3m-account-settings-view/index.ts

```diff
@@ -183,6 +188,53 @@ export class W3mAccountSettingsView extends LitElement {
     `
   }
 
+  private togglePreferredAccountBtnTemplate() {
+    const networkEnabled = NetworkController.checkIfSmartAccountEnabled()
+    const type = StorageUtil.getConnectedConnector()
+    const emailConnector = ConnectorController.getEmailConnector()
+
+    if (!emailConnector || type !== 'EMAIL' || !networkEnabled) {
+      return null
+    }
+
+    const preferredAccountType = W3mFrameHelpers.getPreferredAccountType()
+    const text =
+      preferredAccountType === 'smartAccount'
+        ? 'Switch to your EOA'
+        : 'Switch to your smart account'
+
+    return html`
+      <wui-list-item
+        variant="icon"
+        iconVariant="overlay"
+        icon="swapHorizontalBold"
+        iconSize="sm"
+        ?chevron=${true}
+        ?loading=${this.loading}
+        @click=${this.changePreferredAccountType.bind(this)}
+        data-testid="account-toggle-preferred-account-type"
+      >
+        <wui-text variant="paragraph-500" color="fg-100">${text}</wui-text>
+      </wui-list-item>
+    `
+  }
+
+  private async changePreferredAccountType() {
+    const smartAccountEnabled = NetworkController.checkIfSmartAccountEnabled()
+    const preferredAccountType = W3mFrameHelpers.getPreferredAccountType()
+    const accountTypeTarget =
+      preferredAccountType === 'smartAccount' || !smartAccountEnabled ? 'eoa' : 'smartAccount'
```

Comment:
What about having a tupple about the account types and use them everywhere we need? Seems like we are using hard coded strings in different places and this might bring some depth when we do some typos. 

Instead a using a tupple would help us to prevent it I believe:

```js
const AccountTypes = {
   smartAccount: "smartAccount",
   EOA: "eoa"
}
```

```suggestion
      preferredAccountType === AccountTypes.smartAccount || !smartAccountEnabled ? AccountTypes.EOA : AccountTypes.smartAccount
```